qa/common/util.sh

```diff
@@ -177,10 +177,10 @@ function run_server () {
         if command -v gdb; then
           GDB_LOG="gdb_bt.${SERVER_PID}.log"
           echo -e "=== WARNING: SERVER FAILED TO START, DUMPING GDB BACKTRACE TO [${PWD}/${GDB_LOG}] ==="
-          # Dump backtrace log for quick analysis
-          gdb -batch -ex "thread apply all bt" -p "${SERVER_PID}" 2>&1 | tee "${GDB_LOG}"
+          # Dump backtrace log for quick analysis. Allow these commands to fail.
+          gdb -batch -ex "thread apply all bt" -p "${SERVER_PID}" 2>&1 | tee "${GDB_LOG}" || true
```

Comment:
Hi @rmccorm4 
Using pipe within condition making `true` statement not reachable, and I start facing issue during the execution of test scenario. 
I going to reuse this condition with following modification in separate branch: 
```
gdb -batch -ex "thread apply all bt" -p "${SERVER_PID}" 2>&1 >> "${GDB_LOG}" || true
```